docs/config.md

```diff
@@ -109,18 +109,18 @@ For a Robin-Robin coupling, we need to write and read both of `Sink-Temperature`
 ```c++
 readData
 (
-  Sink-Temperature          // e.g. Sink-Temperature-Solid
-  Heat-Transfer-Coefficient // e.g. Heat-Transfer-Coefficient-Solid
+  Sink-Temperature          // e.g., Sink-Temperature-Solid
+  Heat-Transfer-Coefficient // e.g., Heat-Transfer-Coefficient-Solid
 );
 
 writeData
 (
-  Sink-Temperature          // e.g. Sink-Temperature-Fluid
-  Heat-Transfer-Coefficient // e.g. Heat-Transfer-Coefficient-Fluid
+  Sink-Temperature          // e.g., Sink-Temperature-Fluid
+  Heat-Transfer-Coefficient // e.g., Heat-Transfer-Coefficient-Fluid
 );
 ```
 
-For fluid-structure interaction, `writeData` can be `Force` or `Stress`, where `Stress` is essentially a force vector scaled by the cell face in spatial coordinates (with any postfix), thus, a conservative quantity as well. `readData` can be `Displacement` and `DisplacementDelta` (with any postfix). `DisplacementDelta` refers to the last coupling time step, which needs to considered in the case of subcycling. Structure solvers (such as solids4Foam) can also write `Displacement` and read `Force`.
+For fluid-structure interaction, `writeData` can be `Force` or `Stress`, where `Stress` is essentially a force vector scaled by the cell face in spatial coordinates (with any postfix), thus, a conservative quantity as well. `readData` can be `Displacement` and `DisplacementDelta` (with any postfix). `DisplacementDelta` refers to the last coupling time step, which needs to considered in the case of subcycling. Structure solvers (such as solidFoam or solids4Foam) can also write `Displacement` and read `Force`.
```

Comment:
```suggestion
For fluid-structure interaction, `writeData` can be `Force` or `Stress`, where `Stress` is essentially a force vector scaled by the cell face in spatial coordinates (with any postfix), thus, a consistent quantity. `readData` can be `Displacement` and `DisplacementDelta` (with any postfix). `DisplacementDelta` refers to the delta of the last coupling time step, which needs to considered in the case of subcycling. Structure solvers (such as solidFoam or solids4Foam) can also write `Displacement` and read `Force`.
```
I'm pretty sure that `Stress` should be mapped consistent. I would also rewrite this a bit: either say initially which fields are supported for Fluid and which for Solid or just list all of them under 'reading' and 'writing'. Right now, it sounds like we had already documentation for a Fluid participant and just tried to add a sentence about the Solid side, which came with new capabilities.

Reply:
I changes this paragraph to an itemized list, I think this makes it clearer.